src/ui/public/chrome/api/controls.js

```diff
@@ -19,6 +19,10 @@ export default function (chrome, internals) {
    * @return {chrome}
    */
   chrome.setVisible = function (display) {
+    if (!def) {
```

Comment:
Not sure if this is worth changing in this PR, but the legacy of controls.js and the `def` variable are confusing. To me, a default is a value that can be changed. Since the method setVisible is provided, I would assume that default could be changed. And - to an external consumer, how do setVisible and setVisibleDefault relate to one-another.
 Maybe `def` could be changed to `noChromeMode` and `setVisibleDefault` could be renamed `setNoChromeMode`. That way, the behavior of the variable would be better reflected in the name.